wallet/src/App.vue

```diff
@@ -10,36 +10,42 @@
         {{ option.value }}
       </option>
     </select>
-    <p v-if="walletAddress">Address: {{ walletAddress }}</p>
-    <button
-      v-if="walletAddress"
-      type="button"
-      class="walletButton"
-      @click="handleLogOut"
-    >
-      Disconnect {{ selectedWallet }}
-    </button>
+    <div class="header" v-if="walletAddress">
+      <p>Address: {{ walletAddress }}</p>
+      <button type="button" class="walletButton" @click="executeTx(10e6)">
+        Send 10 Algo
+      </button>
+      <button type="button" class="walletButton" @click="executeAppTx(189)">
+        Application Call
```

Comment:
Can you move this application ID to constants so that it can be easily configured by the user

Reply:
ok. Moved to constant fil.